src/app/components/loaders/bitcoin-account-loader.tsx

```diff
@@ -33,7 +33,7 @@ export function BitcoinNativeSegwitAccountLoader({
 
   const signer = useNativeSegwitSigner(properIndex);
 
-  if (!signer || !isBitcoinEnabled) return fallback;
+  if (!signer || !isBitcoinEnabled) return fallback ? fallback : null;
```

Comment:
This shouldn't be necessary. fallback can be undefined, and undefined is a legit ReactNode?

Reply:
Yep, thx, meant to revert ...was trying a bunch of things to locate the bug.